plugin/src/test/java/org/opensearch/ml/action/MLCommonsIntegTestCase.java

```diff
@@ -93,15 +94,27 @@
 import org.opensearch.ml.utils.TestData;
 import org.opensearch.plugins.Plugin;
 import org.opensearch.search.builder.SearchSourceBuilder;
-import org.opensearch.test.OpenSearchIntegTestCase;
+import org.opensearch.test.ParameterizedStaticSettingsOpenSearchIntegTestCase;
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
 import com.google.gson.Gson;
 
-public class MLCommonsIntegTestCase extends OpenSearchIntegTestCase {
+public class MLCommonsIntegTestCase extends ParameterizedStaticSettingsOpenSearchIntegTestCase {
     private Gson gson = new Gson();
 
+    /**
+     * set ML_COMMONS_SYNC_UP_JOB_INTERVAL_IN_SECONDS to 0 to disable ML_COMMONS_SYNC_UP_JOB
+     * the cluster will be pre-created with the settings at startup
+     */
+    public MLCommonsIntegTestCase() {
+        super(Settings.builder().put(ML_COMMONS_SYNC_UP_JOB_INTERVAL_IN_SECONDS.getKey(), 0).build());
```

Comment:
Will this set sync up job interval as 0 for all integ test cases ? If yes, we should avoid this

Reply:
Here are the implementations of this base class, I am not quite sure they will depends on `MLSyncUpCron`
![image](https://github.com/opensearch-project/ml-commons/assets/113890546/4acc7bdf-0f2d-4f77-a2fc-8cc28e9ea748)

Reply:
Hard code this in the base class looks risky. Can we add this to the child class which is flaky?

Reply:
sure, move the settings to `PredictionITTests`, will add it one by one if we have other similar in future.